File: CPP/kalkulator.cpp
/*
 * kalkulator.cpp
 */


#include <iostream>
using namespace std;

void dodaj(){
    int a, b;
    cout << "Podaj dwie liczby: " << endl;
    cin >> a >> b;
    cout << "Suma: " << a + b << endl;
}
void odejmij(){
    int a, b;
    cout << "Podaj dwie liczby: " << endl;
    cin >> a >> b;
    cout << "Różnica: " << a - b << endl;
}
void mnozenie(){
    int a, b;
    cout << "Podaj dwie liczby: " << endl;
    cin >> a >> b;
    cout << "Iloczny: " << a * b << endl;
}
void dzielenie(){
    int a, b;
    cout << "Podaj dwie liczby: " << endl;
    cin >> a >> b;
    cout << "Iloraz: " << a / b << endl;
}

int main(int argc, char **argv)
{
	char operacja;
    cout << "Jakie działanie chcesz wykonać (+, -, *, /)?" << endl;
    cin >> operacja;
    switch(operacja){
        case '+':
            dodaj();
        break;
        case '-':
            odejmij();
        break;
        case '*':
            mnozenie();
        break;
        case '/':
            dzielenie();
        break;
        default:
            cout << "Nie rozumiem!" << endl;
        break;
    }
	return 0;
}
 

File: python/kalkulator.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  kalkulator.py

wynik = 0
znak = ' '   

 
def pobierzZnak():
    global znak 
    znak = (input('Podaj jakie działanie chcesz wykonać: +, - , *, / : '))
    return 0
    
def pobierzLiczbe():
    liczba1 = int(input('Podaj 1 liczbę: '))
    return liczba1
    
def pobierzLiczbe2():    
    liczba2 = int(input('Podaj 2 liczbę: '))
    return liczba2
    
def dodaj(liczba1, liczba2):
    global wynik
    wynik = liczba1 + liczba2

def odejmij(liczba1, liczba2):
    global wynik
    wynik = liczba1 - liczba2

def iloczyn(liczba1, liczba2):
    global wynik
    wynik = liczba1 * liczba2


def iloraz(liczba1, liczba2):
    return liczba1 / liczba2

def drukujWynik():
    print('Wynik: ', wynik)
        

def main(args):
    znak = pobierzZnak()
    liczba1 = pobierzLiczbe()
    liczba2 = pobierzLiczbe2()
    if znak == '+':
        dodaj(liczba1, liczba2)
        print(wynik)
    elif znak == '-':
        odejmij(liczba1, liczba2)
        print(wynik)
    elif znak == '*':
        iloczyn(liczba1, liczba2)
        print(wynik)
    elif znak == '/':
        iloraz(liczba1, liczba2)
        print(wynik)
    else:
        print('Błędny znak!')
        
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
